package.json

```diff
@@ -88,6 +88,7 @@
     "jest-junit": "^5.0.0",
     "jest-localstorage-mock": "^2.3.0",
     "json-pretty": "^0.0.1",
+    "keytar": "4.4.1",
```

Comment:
This is already defined in `app/package.json`. Please update it there rather than add a new `dependency` at the root.